lab1_task1b.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE HTML>
 <html>
```

Comment:
Set the lang attribute.
Check this doc out for the Minimal HTML5 Document:
https://www.sitepoint.com/a-minimal-html-document-html5-edition/